src/model/chat.ts

```diff
@@ -136,6 +136,10 @@ export class ChatModel<
 
         const delta = value.choices[0].delta;
 
+        if (Object.keys(chunk).length === 0) {
```

Comment:
just curious whats this case?

Reply:
it's when the first chunk is received. chunk is initialized to `{}` above.